src/backend/api/Fusion.Resources.Domain/Extensions/FusionPersonProfileExtensions.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Fusion.Integration.Profile;
+using Fusion.Resources.Database.Entities;
+
+namespace Fusion.Resources.Domain
+{
+    public static class FusionPersonProfileExtensions
+    {
+        public static DbAzureAccountStatus GetDbAccountStatus(this FusionPersonProfile profile)
+        {
```

Comment:
could try the new pattern here.. 
```
return profile.InvitationStatus switch {
  InvitationStatus.Accepted => DbAzureAccountStatus.Available,
  InitationStatus.Pending => DbAzureAccountStatus.InviteSent,
  _ => DbAzureAccountStatus.NoAccount
};
```

But i think we are forgetting that the `InvitationStatus` property can be null if i recall, so we need to add a null check aswell. 

Maybe 
```
if (AzureUniqueId.HasValue && InvitationStatus == null) => return Available 

return switch .. 
```

There is nothing preventing them from adding a "consultant" as personnel.

Reply:
Ok, will fix this as proposed